db/src/db.ts

```diff
@@ -131,7 +131,7 @@ export class DB extends EventEmitter {
       const val = await this.db.get(key);
       return JSON.parse(val.toString());
     } catch (err) {
-      if (err.name === 'NotFoundError') {
+      if (err.notFound) {
```

Comment:
Why is this change needed if db intended to be removed? A different `getWithMeta` uses `NotFoundError`...

Reply:
True.  This is actually the even better way to do it, so changing _that_ one.